load-test/core-api/README.md

```diff
@@ -19,13 +19,13 @@ connect_rate_limit: str = "10000/second"
 ### Example 1
 Use the provided FHIR resource files to test LinuxForHealth connect /fhir and /data endpoints:
 ```shell
-cd connect/load-test
+cd connect/load-test/core-api
```

Comment:
Current load tests moved to load-test/core-api to support multiple sets of load tests for different purposes.